### PR TITLE
Render html from monitor

### PR DESCRIPTION
**Pull request by deads2k**
this will give us timelines for *prior* to installation.

### CODE REVIEW

File: cmd/openshift-tests/openshift-tests.go [hunk 1]
```diff
@@ -109,7 +111,7 @@ func main() {
 }
 
 func newRunMonitorCommand() *cobra.Command {
```
Comment (DennisPeriquet) [inline cmd/openshift-tests/openshift-tests.go:113]: I think you renamed this to `monitor_command.NewRunMonitorCommand` so this code is probably no longer needed (it's definitely not called).

Same applies to `monitor_command.NewMonitorCommand` and `newMonitorCommand`.

File: pkg/cmd/monitor_command/run_monitor_command.go [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+		if err := os.MkdirAll(eventDir, os.ModePerm); err != nil {
+			fmt.Printf("Failed to create monitor-events directory, err: %v\n", err)
+			return err
+		}
+
+		if err := monitor.WriteEventsForJobRun(eventDir, recordedResources, recordedEvents, timeSuffix); err != nil {
+			fmt.Printf("Failed to write event data, err: %v\n", err)
+			return err
+		}
+		if err := monitor.WriteTrackedResourcesForJobRun(eventDir, recordedResources, recordedEvents, timeSuffix); err != nil {
```
Comment (DennisPeriquet) [inline pkg/cmd/monitor_command/run_monitor_command.go:152]: Hm..., you are now calling `WriteTrackedResourcesForJobRun` here.  I see it's also called [here](https://github.com/openshift/origin/blob/88793278dda83b5fd5703dc8eea329f806478c20/pkg/test/ginkgo/options_monitor_events.go#L75)
Will the call on line 148 overwrite what the call in line 75 already generated?

Comment (deads2k) [reply pkg/cmd/monitor_command/run_monitor_command.go:152]: > Hm..., you are now calling `WriteTrackedResourcesForJobRun` here. I see it's also called [here](https://github.com/openshift/origin/blob/88793278dda83b5fd5703dc8eea329f806478c20/pkg/test/ginkgo/options_monitor_events.go#L75)
> Will the call on line 148 overwrite what the call in line 75 already generated?

those are two different types I think.  I'd be happy to see them collapsed into one, but I don't think we need that in order to get value here.

File: cmd/openshift-tests/openshift-tests.go (hunk 1 above)
Comment (deads2k) [reply cmd/openshift-tests/openshift-tests.go]: fixed

**Pull request merged**